src/index.js

```diff
@@ -0,0 +1,86 @@
+import './style.css';
+import todoTasks from './modules/taskDB';
+import {
+  addTask,
+  addNewTask,
+  deleteTask,
+  toggleEditMode,
+  saveToLocalStorage,
+} from './modules/addremove';
+
+const listDiv = document.getElementById('list');
+const clearBtn = document.getElementById('clear-completed');
+const inputField = document.getElementById('newtask');
+
+// Function to build the tasks on the screen
+const buildTask = () => {
+  listDiv.innerHTML = ''; // Clear existing tasks before building
+  todoTasks.forEach((task) => {
+    addTask(task.description, task.index, task.completed);
+  });
+  // Get checkboxes after building the tasks
+  const checkboxes = document.querySelectorAll('.check-box');
+  // Add event listener for the 'change' event on the checkboxes
+  checkboxes.forEach((checkbox) => {
+    checkbox.addEventListener('change', () => {
+      const index = Number(checkbox.id.split('-')[0]);
+      const status = todoTasks[index - 1].completed;
+      todoTasks[index - 1].completed = !status;
+    });
```

Comment:
- [ ] According to [this](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md#:~:text=Store%20the%20updated%20array%20of%20items%20in%20local%20storage%2C%20so%20the%20user%20gets%20the%20correct%20list%20of%20values%20after%20the%20page%20reloads%20(which%20means%20that%20any%20changes%20made%20to%20the%20list%20should%20be%20preserved).) requirement, please make sure that the todos status changes (completed or not completed), are preserved in the local storage and when the user reloads the page, they're able to see the updated data presented with a checkmark next to completed tasks.